vtex-apps/product-summary
# Fix buy button adding to minicart when you have more than one SKU

#### What is the purpose of this pull request?
When you click on the `BuyButton` and the product has more than one SKU you should go to the Product page instead of add to the mini-cart

#### What problem is this solving?
`BuyButton` is adding the first SKU of the product when you click on it and you have more than one SKU

#### How should this be manually tested?
Go to the [workspace](https://summarybuybutton--storecomponents.myvtex.com/classic?_q=classic&map=ft) and click on the `BuyButton`. You should go to the product page instead of adding to the mini-cart.

#### Types of changes
